core/drivers/firewall/stm32_rifsc.c

```diff
@@ -370,7 +370,7 @@ static TEE_Result stm32_rifsc_dt_conf_rimu(const void *fdt, int node,
 		return TEE_ERROR_OUT_OF_MEMORY;
 
 	for (i = 0; i < pdata->nrimu; i++) {
-		uint32_t value = fdt32_to_cpu(*conf_list);
+		uint32_t value = fdt32_to_cpu(conf_list[i]);
```

Comment:
I think the commit reference is wrong in the commit message, should be:
`Fixes: cd187630b280 ("drivers: add stm32 RIFSC support")`

Reply:
ah yes, thx